gpiozero/pins/pigpiod.py

```diff
@@ -98,7 +99,7 @@ class PiGPIOPin(Pin):
     GPIO_PULL_UP_NAMES = {v: k for (k, v) in GPIO_PULL_UPS.items()}
     GPIO_EDGES_NAMES = {v: k for (k, v) in GPIO_EDGES.items()}
 
-    def __new__(cls, number, host='localhost', port=8888):
+    def __new__(cls, number, host = os.getenv("PIGPIO_ADDR", 'localhost'), port = os.getenv("PIGPIO_PORT", 8888) ):
```

Comment:
To match the style of the rest of the code, there shouldn't be spaces either side of the `=` characters, i.e.
```python
def __new__(cls, number, host=os.getenv("PIGPIO_ADDR", 'localhost'), port=os.getenv("PIGPIO_PORT", 8888)):
```

@waveform80 I guess theoretically these env-vars could change during the lifetime of a program, so should these default values be 'got' at module-import time, instead of during object-creation time? (I'm not sure what standard / pythonic behaviour is in circumstances like this.)